spec/classes/init_spec.rb

```diff
@@ -1,169 +1,174 @@
 require 'spec_helper'
```

Comment:
I unstaged it, although everything falls into my "unsupported os" bucket now, so CI fails.  Regardless, looks like I've cleaned up all the garbage I accidentally pulled across from the forge and rebased to where you are now.